provelo_custom/views/hr_kanban_views_employees.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="hr_kanban_view_employee_extend" model="ir.ui.view">
+        <field name="name">hr_kanban_view_phone_mail</field>
+        <field name="inherit_id" ref="hr.hr_kanban_view_employees" />
+        <field name="model">hr.employee</field>
+        <field name="arch" type="xml">
+            <field name="work_location" position="after">
+                <li t-if="record.work_email.raw_value">
```

Comment:
is it necessary to test on `record.work_email.raw_value` rather than just `record.work_email` ? I've never see this before (but I dont meddle much with kanban views)

Reply:
We decided to remove the dependency to hr_attendance, so I removed this commit.

Reply:
You removed it **in** this => make sure to squash the commits before merging then :-)

Reply:
And my question remains: is it necessary to test on `field_x.raw_value` rather than just `field_x`?

Reply:
Oups sorry, I misread your question.
Basically I copy-pasted the lines from odoo/addons/hr/hr_view.xml (branch 9.0). 
https://github.com/odoo/odoo/blob/1950b49b86deb44f33b6598a00bebc5b0fc42fa6/addons/account/views/account_invoice_view.xml#L524

It seems to be the norm also in 12.0, checkout:
https://github.com/odoo/odoo/blob/1950b49b86deb44f33b6598a00bebc5b0fc42fa6/addons/hr/views/hr_views.xml#L228